src/types/commons.ts

```diff
@@ -57,6 +57,12 @@ export type SdkMetadata = {
     browser: string
     browser_version: string
   }
+  camera_settings?: {
+    aspect_ratio: number | undefined
+    frame_rate: number | undefined
+    height: number | undefined
+    width: number | undefined
```

Comment:
I believe these could be shortened:

```suggestion
    aspect_ratio?: number
    frame_rate?: number
    height?: number
    width?: number
```